File: Acceso a datos/003-Herramientas de mapeo objeto relacional (ORM)/001-Concepto de mapeo objeto relacional/102-Actividad EVAL/Respuesta.py
import pickle

# CREO ALGUNOS OBJETOS JUGADOR
class Jugador():
  def __init__(self,nombre,apellidos,emails):
    self.nombre = nombre
    self.apellidos = apellidos
    self.emails = emails
    
jugadores = []
for _ in range(0,10):
  jugadores.append(
    Jugador(
      "Agustín",
      "Morcillo Aguado",
      ["info@agus.es","info@agustin.com"]
      )
  )

print(jugadores)

# SERIALIZO LOS OBJETOS JUGADOR A UN FICHERO BINARIO

archivo = open("jugadores.bin",'wb')
pickle.dump(jugadores,archivo)
archivo.close()

# DESERIALIZO LOS OBJETOS JUGADOR DESDE EL FICHERO BINARIO

archivo = open("jugadores.bin",'rb')
jugadores = pickle.load(archivo)
archivo.close()

print(jugadores)

File: Acceso a datos/005-Bases de datos documentales/001-Bases de datos documentales nativas/102-Actividad EVAL/Respuesta.js
use empresadam;

db.clientes.insertMany([
    {
        nombre: "Agustín",
        apellidos: "Morcillo Aguado",
        telefono: "+34 68547859",
        email: "info@agustin.es"
    },
    {
        nombre: "Elena",
        apellidos: "Botezatu",
        telefono: "+34 123654789",
        email: "info@elena.es"
    },
    {
        nombre: "Lilo",
        apellidos: "Morcillo",
        telefono: "+34 987456321",
        email: "info@lilo.es"
    },
    {
        nombre: "Dipsy",
        apellidos: "Morcillo",
        telefono: "+34 654789321",
        email: "info@dipsy.es"
    }
]);

db.clientes.find();